wagtail/admin/templates/wagtailadmin/generic/chooser/chooser.html

```diff
@@ -45,13 +45,39 @@
             {% endblock %}
 
             {% if is_multiple_choice %}
-                <form action="{{ chosen_multiple_url }}" method="GET" data-multiple-choice-form>
+                <form
+                    action="{{ chosen_multiple_url }}"
+                    method="GET"
+                    data-controller="w-count w-bulk" data-action="w-count#count" data-w-count-find-value="input:checked" data-w-count-container-value="[data-multiple-choice-form]"
```

Comment:
```suggestion
                    data-controller="w-count w-bulk" data-action="w-count#count" data-w-count-find-value="input:checked"
```

You shouldn't need the selector, unless you are intentionally trying to grab all forms. By default it will only count items with in the controlled element.